tic-tac-toe-app/styles.css

```diff
@@ -0,0 +1,21 @@
+.header {
```

Comment:
The styles you have created each align with a className referenced in your components. 💯